doc/nrf/releases_and_maturity/migration/migration_sysbuild.rst

```diff
@@ -0,0 +1,660 @@
+.. _child_parent_to_sysbuild_migration:
+
+Migrating from multi-image builds to sysbuild
+#############################################
+
+:ref:`sysbuild` is a build system used in zephyr to configure, build, and flash multiple images as part of a single project.
+It replaces the :ref:`child/parent system for multi-image builds <ug_multi_image>` in |NCS|.
+As the previous system has been deprecated, you must update your existing multi-image build projects to support being built using sysbuild.
+
+The following are the differences in how project configuration is performed in sysbuild compared to child/parent image configuration:
+
+* Sysbuild controls which images are added to a build, instead of the main application.
+* Sysbuild specifies the project configuration for all images, which was previously done in the main application or child applications.
+* Sysbuild controls the packaging of firmware components, instead of the main application.
+* Sysbuild manages some software functionality of image builds, such as the configuration mode of an nRF70-series radio.
+* Sysbuild runs partition manager code (if enabled) and distributes the output information to images.
+
```

Comment:
This is outside the scope of sysbuild as it would be the same thing in child/parent, if e.g. you enable logging on the main image, it will not enable logging on any other image

Reply:
I don't agree that this is outside the scope of sysbuild.

Due to some KConfigs becoming sysbuild KConfigs, we have been forced to create duplicate overlays customizing both sysbuild and application Kconfigs for sample build configurations where, previously, only a single overlay sufficed.

Some kind of solution to allow us to return to a single overlay or config value would be greatly appreciated for simplifying build configuration for these samples.